### PR TITLE
Add view command

### LINKED ISSUE #89: Add display contact card on CLI

### PR DESCRIPTION
**Pull request by JiaYi-Gallium369**
Close #89

### CODE REVIEW

**Comment by ruiming97** [comment]
Overall LGTM! Perhaps can add more tests to cover more lines

**Review by ruiming97 — COMMENTED** [review]
Overall LGTM! Perhaps can add more tests to cover some lines

File: src/main/java/seedu/address/logic/commands/ViewCommand.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    /**
+     * @param index of the person in the filtered person list to edit
+     * @param contactDisplay to display the person
+     */
+    public ViewCommand(Index index, ContactDisplay contactDisplay) {
+        requireNonNull(index);
+        this.index = index;
+        this.contactDisplay = contactDisplay;
+        this.testContactDisplay = new TestContactDisplay();
+    }
```
Comment (charlenetcy) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:41]: Consider adding a test to ensure the ViewCommand instance would be created successfully.

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 2]
```diff
@@ -18,6 +19,7 @@ public class CommandResult {
 
     /** The application should exit. */
     private final boolean exit;
```
Comment (charlenetcy) [inline src/main/java/seedu/address/logic/commands/CommandResult.java:21]:
```suggestion
    private final boolean hasExited;
```

**Review by charlenetcy — COMMENTED** [review]
Overall, I found your code easy to read. Just needs additional testing.

File: src/main/java/seedu/address/logic/commands/ViewCommand.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        this.testContactDisplay = testContactDisplay;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Person> lastShownList = model.getFilteredPersonList();
+
+        if (index.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
```
Comment (leroychiu20) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:59]: Can add a test for this line to ensure that the code works as expected, i.e. CommandException thrown when specified index is more than the size of the list of contacts.

File: src/main/java/seedu/address/logic/parser/ViewCommandParser.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    public ViewCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+
+        Index index;
+
+        try {
+            index = ParserUtil.parseIndex(args);
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ViewCommand.MESSAGE_USAGE), pe);
+        }
```
Comment (leroychiu20) [inline src/main/java/seedu/address/logic/parser/ViewCommandParser.java:30]: Again, adding a test case for these lines would be good to ensure that the ParseException is caught and handled properly.

**Pull request merged**